modules/fedimint-wallet-server/Cargo.toml

```diff
@@ -21,7 +21,6 @@ async-trait = { workspace = true }
 bitcoin = { workspace = true }
 erased-serde = { workspace = true }
 fedimint-api-client = { workspace = true }
-fedimint-bitcoind = { workspace = true }
```

Comment:
Hey, I'm a simple reviewer. I see red lines in `[dependencies]` section, I approve.